src/Sarif/Schemata/sarif-schema.json

```diff
@@ -870,6 +892,26 @@
           }
         },
 
+        "requests": {
+          "description": "An array of external property files containing run.requests arrays to be merged with the root log file.",
+          "type": "array",
+          "minItems": 1,
```

Comment:
>"minItems": 1, [](start = 10, length = 14)

@larrygolding can you verify these settings are in line with the principles established in [https://github.com/oasis-tcs/sarif-spec/issues/270](https://github.com/oasis-tcs/sarif-spec/issues/270)?

Reply:
this change seems to contradict principle #3 in the referenced issue (as does other definitions in the schema)

---
In reply to: [273738544](https://github.com/Microsoft/sarif-sdk/pull/1419#discussion_r273738544) [](ancestors = 273738544)